cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/declarations/RecordTemplateDeclaration.kt

```diff
@@ -35,7 +35,7 @@ import kotlin.collections.ArrayList
 import org.neo4j.ogm.annotation.Relationship
 
 /** Node representing a declaration of a ClassTemplate */
```

Comment:
Do the comments also need to be changed?

Reply:
Ideally we should use the "imported name" syntax in Kolin, e.g. `[ClassTemplate]` instead of `ClassTemplate`, because this way they also get renamed.

Reply:
Updated description